MoltenVK/MoltenVK/GPUObjects/MVKDevice.mm

```diff
@@ -794,6 +794,12 @@
 				divisorProps->maxVertexAttribDivisor = kMVKUndefinedLargeUInt32;
 				break;
 			}
+			case VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_VERTEX_ATTRIBUTE_DIVISOR_PROPERTIES_KHR: {
+				auto* divisorProps = (VkPhysicalDeviceVertexAttributeDivisorPropertiesKHR*)next;
+				divisorProps->maxVertexAttribDivisor = kMVKUndefinedLargeUInt32;
+				divisorProps->supportsNonZeroFirstInstance = VK_TRUE;
+				break;
+			}
```

Comment:
Hmmm...although this is directly related to the one above it, move this to the `KHR` section just above here (and then in alpha order).

Reply:
are they sorted alphabetically, too? Or is it just sections?

Reply:
Just noticed you already answered my question... Fixed.